doc/help/kdb-record.md

```diff
@@ -0,0 +1,144 @@
+# kdb-record(1) -- Session recording for Elektra
+
+## SYNOPSIS
+
+`kdb record <COMMAND>`
+
+Where `<COMMAND>` is one of the following:
+
+- `start [session_name]`: start or resume a recording session.
+- `stop`: stop the active recording session.
+- `reset [session_name]`: remove all logged data from the recording session.
+- `delete [session_name]`: delete the recording session.
+- `mark [--meta meta_name] <key_name>`: marks a key to be used for validation.
+- `list`: list all available recording sessions.
+- `name`: prints the name of the currently active session.
+- `changes [session_name]`: shows a human-readable list of changes.
+- `export  [--replay|--diff] [session_name]`: creates a file with the changes in the session.
+- `import [--disable-validation] <file_name>`: imports files created by the `export` command.
```

Comment:
I am not a big fan of `--force` either but it makes sense to (additionally) support it as people will immediately look for it if something fails for a non-permanent reason. `validation` in specific, however, is misleading, maybe `--disable-assertions` will do?

Reply:
Yes, `--disable-assertions` would be a good compromise, as validation could entail much more, e.g. checking that the file format is correct.

Reply:
I'd suggest `--no-assertions` or `--without-assertions`. The `--disable` sounds a bit like it changes some config somewhere instead of just being a temporary flag.

Reply:
Yes, good point.

We should watch out that this is consistent within the whole tooling. Currently `kdb` is inconsistent, we have --no-newline but --without-elektra and --with-recommends.